smallrye-reactive-messaging-kafka/pom.xml

```diff
@@ -39,6 +39,11 @@
       <groupId>io.opentelemetry</groupId>
       <artifactId>opentelemetry-semconv</artifactId>
     </dependency>
+    
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
```

Comment:
Instead of adding this dependency (which causes an issue in native), I would prefer to have a simple utility method. You only use the `isNotEmpty` which can easily be implemented.

Something like this:

```java
public boolean isNotBlank(String s) {
    return s != null && ! s.trim().isEmpty();
}
```